api/python/cellxgene_census/src/cellxgene_census/_get_anndata.py

```diff
@@ -29,20 +29,20 @@ def get_anndata(
     column_names: Optional[soma.AxisColumnNames] = None,
 ) -> anndata.AnnData:
     """
-    Convience wrapper around ``soma.Experiment`` query, to build and execute a query,
+    Convenience wrapper around :class:`tiledbsoma.Experiment` query, to build and execute a query,
```

Comment:
recommend consistently using `soma.` or `tiledsoma.`

We tend to import as `soma`, but either is fine as long as it is consistent

Reply:
I opted for `tiledbsoma.` as I wasn't sure if `soma` would render correctly since the package is called `tiledbsoma`.